### PR TITLE
Added bruno to the things people have built using Ohm

### PR DESCRIPTION
**Pull request by helloanoop**

### CODE REVIEW

**Comment by helloanoop** [comment]
@pdubroy Would love to have this merged 💛

File: src/pages/_postHomepageFeaturesContent.mdx [hunk 1]
```diff
@@ -8,6 +8,7 @@ Here are some awesome things people have built using Ohm:
 - [Simpletalk][] is an expressive, programmable authoring system inspired by Hypercard and Smalltalk. _(See also the [StrangeLoop talk][simpletalk-talk])._
 - [Wildcard](https://www.geoffreylitt.com/wildcard/), a browser extension that empowers anyone to modify websites to meet their own specific needs, uses Ohm for its spreadsheet formulas.
 - [JAMScript][] is a programming language for edge-based IoT applications.
+- [Bruno][] is an opensource IDE for exploring and testing api's.
```
Comment (pdubroy) [inline src/pages/_postHomepageFeaturesContent.mdx]:
```suggestion
- [Bruno][] is an open source IDE for exploring and testing APIs.
```

**Comment by pdubroy** [comment]
Happy to. I think it's a great project! Makes me smile whenever it comes up on HN recently :-)

**Comment by pdubroy** [comment]
Just deployed. GH pages usually takes a few mins to update; let me know if you don't see it on there soon.

**Pull request merged**